src/app/GitUI/UserControls/FileStatusList.cs

```diff
@@ -572,274 +558,152 @@ public IEnumerable<FileStatusItem> FirstGroupItems
 
         public void ClearSelected()
         {
-            foreach (ListViewItem item in FileStatusListView.SelectedItems)
-            {
-                item.Selected = false;
-            }
+            FileStatusListView.SelectedNodes = [];
         }
 
         public new void Focus()
         {
-            if (FileStatusListView.Items.Count > 0)
+            if (FileStatusListView.Nodes.Count > 0)
             {
                 if (SelectedItem is null)
                 {
-                    SelectedIndex = 0;
+                    SelectFirstVisibleItem();
                 }
 
                 FileStatusListView.Focus();
             }
         }
 
-        private static (Image? image, string? prefix, string text, string? suffix, int prefixTextStartX, int textWidth, int textMaxWidth)
-            FormatListViewItem(ListViewItem item, PathFormatter formatter, int itemWidth)
+        private static (string? prefix, string text, string? suffix) FormatListViewItem(TreeNode node, PathFormatter formatter, int itemWidth)
         {
-            GitItemStatus gitItemStatus = item.Tag<FileStatusItem>().Item;
-            Image? image = item.Image();
-            int itemLeft = item.Position.X;
+            if (node.Tag is not FileStatusItem fileStatusItem || fileStatusItem.Item.IsRangeDiff)
```

Comment:
The `Tag` is what it is: a `FileStatusItem` but never a `GitItemStatus`.

Reply:
It has to be written like following, which I do not request (but slightly prefer)
```suggestion
            if (node.Tag is not FileStatusItem fileStatusItem || fileStatusItem.Item.IsRangeDiff)
```

Reply:
> It has to be written like following, which I do not request (but slightly prefer)

The suggestion has not been updated.
Why? I am asking because it works well.

Reply:
```suggestion
            if (node.Tag is not FileStatusItem fileStatusItem || fileStatusItem.Item is not GitItemStatus gitItemStatus || gitItemStatus.IsRangeDiff)
```

Reply:
not necessary because

![image](https://github.com/user-attachments/assets/012fb88c-6d63-4a63-853b-c3a41afa6f2d)

Reply:
This is more a concept than a request
fileStatusItem  variable is only used to define gitItemStatus, but gitItemStatus is defined below and fileStatusItem.Item is used "early". (The change removes the define below the return.)
The suggestion defines gitItemStatus where it is checked so I get one less "section" when reading

Reply:
I saw the duplication, too. But I would always ask myself how `Item` could be something different.
In order to get rid of the additional line, I would rather remove the variable `GitItemStatus gitItemStatus` and replace the few occurrences with `fileStatusItem.Item`.